quick_sort.py

```diff
@@ -0,0 +1,62 @@
+def quick_sort(array):
+    '''Input an array, the lowest value, and the highest value. Perform a
+    quicksort and return a sorted list'''
+    return _quick_sort_helper(array, 0, (len(array) - 1))
+
+
+def _quick_sort_helper(array, lo, hi):
+    if lo < hi:
+        partition = _partition(array, lo, hi)
+        _quick_sort_helper(array, lo, (partition - 1))
+        _quick_sort_helper(array, (partition + 1), hi)
+    return array
+
+
+def _partition(array, lo, hi):
+    pivot_index = ((lo + hi)/2)
```

Comment:
watch out here.  You certainly mean to say `(lo + hi) // 2`.  The statement you have is *not* the same.